.github/workflows/release.yml

```diff
@@ -22,21 +21,25 @@ jobs:
         with:
           node-version-file: '.nvmrc'
           cache: npm
-      - name: install
-        run: npm ci || npm install
-      - run: npm test
-      - uses: fregante/daily-version-action@v1
+      - run: npm ci
+      - name: Test and build
+        run: npm test
+      - uses: fregante/daily-version-action@v2
         name: Create tag if necessary
-        id: daily-version
-      - if: steps.daily-version.outputs.created
+      - name: Update extension’s meta
+        run: npx dot-json@1 $DIRECTORY/manifest.json version $DAILY_VERSION
+      - uses: actions/upload-artifact@v3
+        with:
+          name: built
+          path: ${{ env.DIRECTORY }}
+      - if: env.DAILY_VERSION_CREATED
         name: Create release
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        working-directory: ${{ env.DIRECTORY }}
         run: |
-          cd distribution
           zip -r refined-github-local-install.zip *
-          gh release create "${{ steps.daily-version.outputs.version }}" --generate-notes refined-github-local-install.zip
-
+          gh release create "$DAILY_VERSION" --generate-notes refined-github-local-install.zip
 
   Submit:
     needs: Version
```

Comment:
The condition below is fired even if `needs.Version.outputs.created` isn't set. The intention was to always attempt to submit to the stores even on re-runs, but I don’t think this ever worked correctly and should be dropped for now.